example/finalize.c

```diff
@@ -12,8 +12,8 @@ const int iterations = 100000;
 int live_count = 0;
 
 void finalize(void* data) {
-  int i = (int)data;
-  if (i % (iterations / 10) == 0) printf("Finalizing #%d...\n", i);
+  int i = (intptr_t)data;
```

Comment:
Shouldn't the declared type be intptr_t as well, then?

Reply:
Yes of course. _Thank you._